commit b6c31e0c77a74458fcfa7938a0fbc3b55c7476fc
Author: Matias De lellis <mati86dl@gmail.com>
Date:   Thu Jun 22 21:08:46 2017 -0300

    D'Oh!. Fix missing function due typo...

    Update deps to grilo 0.3.

Conflicts:
	configure.ac
	plugins/ampache/pragha-ampache-plugin.c
	plugins/cdrom/pragha-cdrom-plugin.c
	src/pragha-backend.c
	  (only backported the DLNA plugin changes)

diff --git a/plugins/dlna-renderer/pragha-dlna-renderer-plugin.c b/plugins/dlna-renderer/pragha-dlna-renderer-plugin.c
index 9081ea11e9def9ea..2cedb36c4a9b0d94 100644
--- a/plugins/dlna-renderer/pragha-dlna-renderer-plugin.c
+++ b/plugins/dlna-renderer/pragha-dlna-renderer-plugin.c
@@ -154,10 +154,10 @@ pragha_dlna_renderer_append_source (GList     *list,
 
 		media = GRL_MEDIA (media_iter->data);
 
-		if (GRL_IS_MEDIA_BOX (media)) {
+		if (grl_media_is_container (media)) {
 			list = pragha_dlna_renderer_append_source (list, source, media);
 		}
-		else if (GRL_IS_MEDIA_AUDIO (media)) {
+		else if (grl_media_is_audio (media)) {
 			list = pragha_dlna_renderer_append_media (list, media);
 		}
 		pragha_process_gtk_events ();
@@ -236,7 +236,7 @@ pragha_plugin_activate (PeasActivatable *activatable)
 	grl_init (NULL, NULL);
 
 	registry = grl_registry_get_default ();
-	if (!grl_registry_load_plugin_by_id (registry, "grl-dleyna", &error)) {
+	if (!grl_registry_activate_plugin_by_id (registry, "grl-dleyna", &error)) {
 		g_print ("Failed to load plugins: %s\n\n", error->message);
  	}
 
@@ -281,4 +281,4 @@ pragha_plugin_deactivate (PeasActivatable *activatable)
 	pragha_menubar_remove_action (priv->pragha, "pragha-plugins-placeholder", "search-dlna");
 
 	grl_deinit ();
-}
\ No newline at end of file
+}
